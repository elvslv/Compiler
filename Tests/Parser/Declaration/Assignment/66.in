type tarr = array[1..10] of integer;
function func: tarr; forward;
var i: integer;
begin
	i := func[5];
end.