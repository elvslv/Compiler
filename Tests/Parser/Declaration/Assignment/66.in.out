type TARR = array [const 1..const 10] of INTEGER;
function FUNC: TARR;var I : INTEGER;

:=              
|               
|___I           
|               
|___[]          
    |           
    |___()      
    |   |       
    |   |___FUNC
    |           
    |___5       
